src/services/sync/repo/user.repo.js

```diff
@@ -97,8 +97,9 @@ const updateAccount = async (userId, account) =>
 
 const updateUserAndAccount = async (userId, changedUser, changedAccount) => {
 	await checkUpdate(changedUser.email, userId);
-	changedUser.roles = await resolveUserRoles(changedUser.roles);
-
+	if ('roles' in changedUser) {
```

Comment:
```suggestion
	if (changedUser.roles) {
```

Reply:
I have found the way i used in https://stackoverflow.com/questions/11040472/how-to-check-if-object-property-exists-with-a-variable-holding-the-property-name

Reply:
it is also correct